src/app/pages/MediaArticlePage/utils/checkIsLiveMedia/index.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  AresMediaBlock,
+  AresMediaMetadataBlock,
+} from '#app/components/MediaLoader/types';
+import filterForBlockType from '#app/lib/utilities/blockHandlers';
+import { OptimoBlock } from '#app/models/types/optimo';
+
+const checkIsLiveMedia = (blocks: OptimoBlock[]) => {
+  const mediaBlocks = blocks.filter(
+    block => block.type === 'video' || block.type === 'audio',
+  );
+
+  if (!mediaBlocks || mediaBlocks?.length !== 1) return false;
```

Comment:
Could there ever be a case where there are 2 pieces of media on the live tv page - one of them a live stream, one of them not, or 2 live streams (very unlikely)? If so, are we just best checking for any live media in the data, and returning true? Or is that overkill / premature optimisation / trying to handle a scenario which would never occur IRL?

Reply:
May be slight overkill. There could be a scenario whereby we have a live stream in an article along with normal media, so in that case we _do_ want to show the timestamp.